dict/prh-idiomatic-usage.yml

```
@@ -206,7 +206,12 @@ rules:
         to: 皆さま
   - expected: 振り込み
     pattern:
-      - 振込
+      - (?<!給与)振込(?<!口座)$
```

```suggestion
      - (?<!給与)振込(?!口座)
```
口座の部分が「否定・後読み」になっていたのですが、「否定・先読み」が意図したものかもです。

また、行末か否かは検知には影響しないので、$はトルが良さそうです。